### ISSUE TITLE
how to setup zergpool

### ISSUE THREAD

**Issue by GkyEla** [UnamSanctam/SilentCryptoMiner#1988]
-a kawpow -o stratum+tcp://kawpow.eu.mine.zergpool.com:3638 -u walletadress --timeout 120 -p  c=XMR

guys iwant to setup kawpow zergpool for silentcryptominer what should i fill the blanks with
also i chose kawpow from create new miner part is that correct?

![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/60583610/12b900f7-6ce2-41c0-a182-292504c44d0a)

**Comment by GkyEla** [comment]
thanks for helping also if i want to see computername what should i write
and i chouse kawpow in create new miner section right?

**Comment by UnamSanctam** [comment]
Yes KawPow is correct, if you want to add the computer name for Zergpool then you can change the "Password" to this: `c=XMR,ID={COMPUTERNAME}`

**Comment by GkyEla** [comment]
Thank you so much Unam for everything and my last question is 
![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/60583610/dde1114e-9b4b-4938-83fb-5a13c1eef7e7)
what is the difference standard and ssl/tls also which one do you prefer?

**Comment by UnamSanctam** [comment]
Standard is just a normal unencrypted connection to the pool, with SSL/TLS the connection will instead be encrypted. Preferably SSL/TLS should be used if possible, if you do want to use that on Zergpool then change the port to `13638` instead, so: `kawpow.eu.mine.zergpool.com:13638`
